### PR TITLE
CP-12326: move guest-packages.hg/utils into xe-guest-utilities.git

### PR DESCRIPTION
**Pull request by phusl**
Signed-off-by: Phus Lu <phus.lu@citrix.com>

### CODE REVIEW

**Comment by Zhengchai** [comment]
thank you @phusl - looks good to me.
A minor point is that -- we may need to delete the old license file under the mk file, while use the new LICENSE file for both internal packaging and outside packing.

**Comment by phusl** [comment]
Thank you @Zhengchai, I updated the PR with `commit --amend`, now the original COPYING/COPYING.LGPL was replaced with LICENSE.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Copyright (c) 2015, Citrix Systems
+All rights reserved.
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```
Comment (robertbreker) [inline LICENSE]: Could we please add line breaks to get below 80 chars per line?
E.g. https://github.com/xenserver/xscontainer/blob/master/LICENSE with the updated copyright year.

File: mk/xe-guest-utilities.spec.in [hunk 2]
```diff
@@ -0,0 +1,124 @@
+# -*- rpm-spec -*-
+
+Summary: @BRAND_GUEST@ Monitoring Scripts
+Name: xe-guest-utilities
+Version: @VERSION@
+Release: @RELEASE@
+License: GPL/LGPL
```
Comment (robertbreker) [inline mk/xe-guest-utilities.spec.in]: I think this should be BSD.

File: mk/Makefile.deb [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+	cp $(MY_OBJ_DIR)/$(PACKAGE)_$(VERSION)-$(RELEASE).dsc $(MY_OUTPUT_DIR)/$(DEBPOOLDIR)
+	cp $(MY_OBJ_DIR)/$(PACKAGE)_$(VERSION)-$(RELEASE).tar.gz $(MY_OUTPUT_DIR)/$(DEBPOOLDIR)
+	set -xe ; for arch in $(ARCHS) ; do \
+		( cd $(SOURCEDIR) && dpkg-buildpackage -Zgzip -b -a$${arch} -us -uc ) ; \
+		cp $(MY_OBJ_DIR)/$(PACKAGE)_$(VERSION)-$(RELEASE)_all.deb $(MY_OUTPUT_DIR)/$(DEBPOOLDIR) ; \
+	done
+	touch $@
+
+$(MY_SOURCES)/MANIFEST: $(MY_SOURCES_DIRSTAMP)
+	( echo "$(COMPONENT) GPL file $(MY_OUTPUT_DIR)/$(DEBPOOLDIR)/$(PACKAGE)_$(VERSION)-$(RELEASE).dsc" ; \
```
Comment (robertbreker) [inline mk/Makefile.deb]: I think this should be BSD.

File: mk/Makefile.deb [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+	  echo "$(COMPONENT) GPL file $(MY_OUTPUT_DIR)/$(DEBPOOLDIR)/$(PACKAGE)_$(VERSION)-$(RELEASE).tar.gz" ; \
```
Comment (robertbreker) [inline mk/Makefile.deb]: ditto.

File: mk/Makefile.rpm [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+	)>$@
+
+$(RPM_BUILT_COOKIE): $(RPM_DIRECTORIES) $(RPM_SPECSDIR)/$(SPEC) $(SOURCES)
+	$(RPMBUILD) -bs $(RPM_SPECSDIR)/$(SPEC)
+	set -ex ; for arch in $(ARCHS) ; do \
+		$(RPMBUILD) --target=$${arch} --rebuild $(RPM_SRPMSDIR)/$(SRPM) ; \
+	done
+
+$(MY_SOURCES)/MANIFEST: $(MY_SOURCES_DIRSTAMP)
+	( echo "$(COMPONENT) GPL file $(RPM_SRPMSDIR)/$(SRPM)" ; \
```
Comment (robertbreker) [inline mk/Makefile.rpm]: ditto

File: mk/xe-linux-distribution [hunk 6]
```diff
@@ -0,0 +1,307 @@
+#! /bin/sh
+
+# Copyright (C) 2009  Citrix Systems Inc.
+#
+# This program is free software; you can redistribute it and/or
```
Comment (robertbreker) [inline mk/xe-linux-distribution]: I think this may need updating as well?

**Comment by Zhengchai** [comment]
@robertbreker hi Robert - I have update the comments, could you have a look for the reworked version please ?

**Comment by rdobson** [comment]
@Zhengchai - this looks good to me, as far as I can tell you've address all of @robertbreker's comments.

**Comment by robertbreker** [comment]
Thanks @Zhengchai.

This looks good. I've just got a minor suggestion, if you don't mind. However, if you're in a rush, we can always do this later.
We typically try to abbreviate license names according to the Fedora packaging guidelines at https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#SoftwareLicenses. If you get a chance, you may also want to replace the short name "BSD 2-Clause" with "BSD".
Sorry for not pointing this out earlier!

Best wishes,
Robert

**Comment by Zhengchai** [comment]
Thank you @robertbreker @rdobson Glad to learn this short name. No worries. changed just now. and I am going to merge it now for a build.

**Pull request merged**